superbench/cli/_commands.py

```diff
@@ -23,6 +23,8 @@ def load_command_table(self, args):
             g.command('deploy', 'deploy_command_handler')
             g.command('exec', 'exec_command_handler')
             g.command('run', 'run_command_handler')
+        with CommandGroup(self, 'node', 'superbench.cli._node_handler#{}') as g:
+            g.command('info', 'info_command_handler')
```

Comment:
How can we collect all information from all nodes?

Reply:
it can only be used for one node
the cli is even not installed on any compute node

Reply:
Shall we add an ansible task for this? Maybe we can do it in future. I just think we can leverage this to check the configuration.

Reply:
it's not aligned with current design (cli is only run in container, host does not install cli, but the collection needs to be run on host)
we can add such a feature later